oas_apivideo.yaml

```diff
@@ -4319,27 +4333,55 @@ paths:
               schema:
                 $ref: '#/components/schemas/live-stream'
               examples:
-                response:
-                  value:
-                    liveStreamId: li4pqNqGUkhKfWcBGpZVLRY5
-                    createdAt: '2020-07-29T10:45:35.000Z'
-                    updatedAt: '2020-07-29T10:45:35.000Z'
-                    streamKey: cc1b4df0-d1c5-4064-a8f9-9f0368385135
-                    name: Live From New York
-                    public: true
-                    record: true
-                    broadcasting: false
-                    assets:
-                      iframe: '<iframe src="https://embed.api.video/live/li4pqNqGUkhKfWcBGpZVLRY5" width="100%" height="100%" frameborder="0" scrolling="no" allowfullscreen=""></iframe>'
-                      player: 'https://embed.api.video/live/li4pqNqGUkhKfWcBGpZVLRY5'
-                      hls: 'https://live.api.video/li4pqNqGUkhKfWcBGpZVLRY5.m3u8'
-                      thumbnail: 'https://cdn.api.video/live/li4pqNqGUkhKfWcBGpZVLRY5/thumbnail.jpg'
+                live-stream-response-example:
+                  $ref: '#/components/examples/live-stream-response-example'
```

Comment:
Is this a Readme ref? just making sure that Doctave stuff are not here :D

Reply:
The response is missing?

Reply:
This is not Doctave, only Readme stuff. You can find the response example [at this line](https://github.com/apivideo/api.video-api-specification/pull/54/files#diff-70177cb6616d8d9dee50ca758bc3ae77ac956d90bc54047c8e688575611b39f8R10997), the rest are just references to the same response.